docs/src/pages/index.js

```diff
@@ -24,14 +22,7 @@ function HomepageHeader() {
   );
 }
 
+import {Redirect} from '@docusaurus/router';
 export default function Home() {
-  const { siteConfig } = useDocusaurusContext();
-  return (
-    <Layout>
-      <HomepageHeader />
-      <main>
-        <HomepageFeatures />
-      </main>
-    </Layout>
-  );
+    return <Redirect to="/docs/general/Introduction/why-askui" />;
 }
```

Comment:
Doesn't this remove the homepage of our documentation?

Reply:
That is correct.

All the tools I know directly link to their "Getting started" or "Introduction" when you navigate to their docs.

Reply:
Not But can't you do that without removing the page altogether? :D 

Btw, I think I have plenty examples for tools where I think that this is not the case:

- https://codecept.io/
- https://playwright.dev/
- https://fluxcd.io/
- https://kubernetes.io/

just to name the view

Reply:
Fair enough. Did not check those 😄 

Those docs seem to use Docusaurus also and the homepage as their landing page.

Our page seems superfluous. You want to get to the docs and the page displayed does not help you in my opinion.
At least we should make it more useful?

Reply:
Agreed, I am fine with having it removed for now as long as everything still works.